cl/lib/tests.py

```diff
@@ -48,20 +49,24 @@ def test_auto_blocking_small_bankr_docket(self) -> None:
         """Do we properly set small bankruptcy dockets to private?"""
         d = Docket()
         d.court = Court.objects.get(pk="akb")
-        blocked, date_blocked = get_blocked_status(d)
-        self.assertTrue(
-            blocked,
-            msg="Bankruptcy dockets with few entries should be blocked.",
+        blocked, date_blocked = async_to_sync(get_blocked_status)(d)
+        # self.assertTrue(
+        #     blocked,
+        #     msg="Bankruptcy dockets with few entries should be blocked.",
+        # )
```

Comment:
Not sure the right way to fix this one test.

Reply:
I think the suggestion I made for looking up via the Court table instead of the Docket table should fix this. The problem was that `d = Docket()` doesn't save it to the DB, so your `aexists` call was returning false — it didn't yet.

Reply:
> I think the suggestion I made for looking up via the Court table instead of the Docket table should fix this.

Changed to that but this is still failing it seems.

Reply:
nvm, didn't apply that change correctly, seems to be working now